### PR TITLE
Minor changes in .spec file

### PR DESCRIPTION
**Pull request by sohankunkerkar**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by sohankunkerkar** [comment]
Passing an RPM build(https://brewweb.engineering.redhat.com/brew/taskinfo?taskID=22845723)

File: kubefedctl.spec [hunk 1]
```diff
@@ -115,22 +115,10 @@ install -d %{buildroot}%{_bindir}
 
 install -p -m 755 $RPM_BUILD_DIR/go/src/sigs.k8s.io/kubefed/bin/kubefedctl %{buildroot}%{_bindir}/kubefedctl
 
-# EXAMPLE: Install tests
```
Comment (abhat) [inline kubefedctl.spec]: Can you get rid of the commented code?

**Pull request merged**